### PR TITLE
cleanup-- no functional changes

### PR DESCRIPTION
**Pull request by krancour**
This is just a little bit of cleanup.

The main change here is we're shying away from package-scoped variables in favor of a more object-oriented approach.

To be fully transparent about this, some Gophers would frown on this, claiming that Go isn't an OO language. However, Go's own creators acknowledge the language possesses OO features, and for good reason. My rationale for this change is that values of package level variables are too easily _corruptable_. Consider what could happen with package level variables as our test coverage expands. Are their values reset between tests? I can never recall, but with this approach, it wouldn't matter if they weren't. An even better question-- if the tests cases were executed concurrently (which we can do), would we have race conditions with some tests setting metrics and then not getting the expected results because another test clobbered the metric in the meantime? These concerns are negated by encapsulating these variables so that each test case could have its _own_ `metricExporter`.

In short, this is generally a _safer_ style that leads to less error-prone code.

### CODE REVIEW

**Comment by willie-yao** [comment]
That makes a lot of sense! I see that this is how it is done in `brig term` as well. LGTM
